index.html

```diff
@@ -1,15 +1,41 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>WDI Radio</title>
-    <script src="js/app.js"></script>
+<html data-ng-app="station">
+<head>
+  <title>WDI Radio</title>
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.15/angular-ui-router.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.0-beta.2/angular-resource.min.js"></script>
 
-  </head>
-  <body>
-    <header>WDI Radio</header>
 
-    Welcome to the World's Best Song App
+  <script src="js/app.js"></script>
+  <script src="js/station/songs.js"></script>
+  <script src="js/station/index.controller.js"></script>
+  <script src="js/station/show.controller.js"></script>
+  <!-- <script src="js/station/new.controller.js"></script> -->
+  <!-- <script src="js/station/welcome.controller.js"></script> -->
+  <script src="js/station/station.factory.js"></script>
+  <script src="js/station/form.directive.js"></script>
```

Comment:
I'd suggest using `songs` for this folder, since the fundamental resource these files are for is songs. If your app had the concept of a station(s), then you'd want a separate station folder.